template/composer.json

```diff
@@ -94,7 +94,7 @@
   },
   "autoload": {
     "psr-4": {
-      "Studiometa\\": "web/wp-content/themes/<?= slug %>/app"
+      "Studiometa\\": "web/wp-content/themes/<% slug %>/app"
```

Comment:
`<%= slug %>` is used throughout the project, i think it's the correct fix. Can you check ?
Could you also update https://github.com/studiometa/create-wordpress-project/blob/develop/saofile.js#L109 ?

Reply:
I can confirm that it should be `<%= slug %>` here 😉

Reply:
My bad, it's solved ! :)